### PR TITLE
Feat/navigation

### PR DESCRIPTION
**Pull request by tonymike99**
Hi team,
I have added the navigation component

[Check only these files](https://github.com/tonymike99/candel-css/pull/11/files?file-filters%5B%5D=.css&file-filters%5B%5D=.html&show-viewed-files=true)

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
Clean understandable code
good to merge

**Review by amartya-rs — APPROVED** [review]
Code looks fine to me. Good to merge.

File: Components/Navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+.brand-container {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    margin: 0 1rem;
+}
+
+.brand-name {
+    background-color: #36096d;
```
Comment (pradipgharge) [inline Components/Navigation/navigation.css:22]: you can use color variable here

File: Components/Navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.navigation {
+    border: 2px solid black;
+    border-radius: 5px;
+}
+
+header {
+    align-items: center;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
```
Comment (pradipgharge) [inline Components/Navigation/navigation.css:10]: you can abstract out the flex properties as a separate class, to avoid repeatition

**Review by pradipgharge — COMMENTED** [review]
Suggested some changes, rest looks good to merge

**Pull request merged**